nixos/jormungandr.nix

```diff
@@ -10,12 +10,13 @@ in {
   options = {
 
     services.jormungandr = {
-      enable = mkOption {
+      enable = mkEnableOption "jormungandr";
+
+      enableExplorer = mkOption {
```

Comment:
this could possibly also use mkEnableOption but I left it as a bool for now. It may become a full set of options in the future as jormungandr config expects it to be a set of a single bool.